JavaScriptTraining/JsTask1_4.js

```diff
@@ -0,0 +1,54 @@
+const listTitle = `
+======================
+現在持っているタスクの一覧
+======================`;
+
+const typeMessage =
+  "「確認、追加、削除、終了」の4つのいずれかを入力してください";
+
+const addMessage = "新しいタスクを追加しました";
+
+const task1 = { category: "掃除", content: "机を片付ける" };
+const task2 = { category: "買い物", content: "牛乳を買う" };
+const task3 = { category: "運動", content: "散歩する" };
+
+let taskList = [task1, task2, task3];
```

Comment:
この場合は下記のようにひとつにしてしまってOKです。
```suggestion
let taskList = [
  { category:  '掃除', content: '机を片付ける'},
  { category:  '買い物', content: '牛乳を買う'},
  { category: '運動', content: '散歩する'}
  ]
```